From 87a85929a46ef76f50a225901ef96ab2ac1eb5cb Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:42:02 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(f004cafa)..lineage-16.0-android-9.0.0_r31(34fe747e) (DO
 NOT SUBMIT)

Android Build Merger (Role) (7):
 - [automerger] DO NOT MERGE: Don't Disable Pkg Verifier When
   Provisioning am: d4c69259fb
 - [automerger] DO NOT MERGE: Don't Disable Pkg Verifier When
   Provisioning am: d4c69259fb am: c7fc0f823a
 - [automerger] DO NOT MERGE: Don't Disable Pkg Verifier When
   Provisioning am: d4c69259fb am: c7fc0f823a am: 60e09ff54d
 - [automerger] DO NOT MERGE: Don't Disable Pkg Verifier When
   Provisioning am: d4c69259fb am: c7fc0f823a am: 60e09ff54d am:
   c564a8fcc4
 - [automerger] DO NOT MERGE: Don't Disable Pkg Verifier When
   Provisioning am: d4c69259fb am: c7fc0f823a am: 60e09ff54d am:
   c564a8fcc4 skipped: 7ee1a034bb
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   6a98a579a8
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   6a98a579a8 skipped: 13c1fc28d6

Bill Yi (7):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   6a98a579a8 skipped: 13c1fc28d6 am: 30bed7b0a2  -s ours
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   6a98a579a8 skipped: 13c1fc28d6 am: 30bed7b0a2  -s ours am: a47787371c
    -s ours
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   6a98a579a8 skipped: 13c1fc28d6 am: 30bed7b0a2  -s ours am: a47787371c
    -s ours am: 9bb8d48894  -s ours

Eric Sandness (1):
 - Don't Disable Pkg Verifier When Provisioning

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

TreeHugger Robot (1):
 - Merge changes from topic "am-628c0ec9-d949-4f88-a8bc-febb80451ed0"
   into oc-dev

android-build-team Robot (3):
 - Snap for 5073521 from 8123730eb899d6bf1c81ea7b6b8a76b78c1e8ee1 to
   pi-qpr2-release
 - Snap for 5115288 from 617e752a405c9893aa0ba116e159f8b8a041f125 to
   pi-qpr2-release
 - Snap for 5130385 from 985eb88f8ebc6b2a9eb62b5de7b119405c3fdd9b to
   pi-qpr2-release

arangelov (8):
 - DO NOT MERGE: Don't Disable Pkg Verifier When Provisioning
 - DO NOT MERGE: Don't Disable Pkg Verifier When Provisioning
 - DO NOT MERGE: Don't Disable Pkg Verifier When Provisioning am:
   84a7ce7e5f
 - DO NOT MERGE: Don't Disable Pkg Verifier When Provisioning am:
   84a7ce7e5f am: 1b1a0066ba
 - [automerger skipped] DO NOT MERGE: Don't Disable Pkg Verifier When
   Provisioning am: 84a7ce7e5f am: 1b1a0066ba am: a78134de32  -s ours
 - Merge changes from topic "am-628c0ec9-d949-4f88-a8bc-febb80451ed0"
   into oc-dev am: 748e7918b2
 - Merge changes from topic "am-628c0ec9-d949-4f88-a8bc-febb80451ed0"
   into oc-dev am: 748e7918b2 am: 55e0c5af8c
 - Merge changes from topic "am-628c0ec9-d949-4f88-a8bc-febb80451ed0"
   into oc-dev am: 748e7918b2 am: 55e0c5af8c am: 3778aa1d0b

Change-Id: Icf1f5afda80f82dd8f0ae352003e41354caadf3d

Change-Id: Ied3aa8f465cee6622359cc00548d2bee9db6e003
---
 res/values-b+sr+Latn/strings.xml | 4 ++--
 res/values-de/strings.xml        | 2 +-
 res/values-mr/strings.xml        | 6 +++---
 res/values-pt-rPT/strings.xml    | 2 +-
 res/values-sr/strings.xml        | 4 ++--
 5 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/res/values-b+sr+Latn/strings.xml b/res/values-b+sr+Latn/strings.xml
index e7b40d06..a1df9df1 100644
--- a/res/values-b+sr+Latn/strings.xml
+++ b/res/values-b+sr+Latn/strings.xml
@@ -48,7 +48,7 @@
     <string name="sure_you_want_to_delete_profile" msgid="6927697984573575564">"Ako nastavite, izbrisaćete sve aplikacije i podatke na ovom profilu."</string>
     <string name="delete_profile" msgid="2299218578684663459">"Izbriši"</string>
     <string name="cancel_delete_profile" msgid="5155447537894046036">"Otkaži"</string>
-    <string name="encrypt_device_text_for_profile_owner_setup" msgid="7828515754696057140">"Da biste podesili profil za Work, uređaj treba da se šifruje. To može da potraje."</string>
+    <string name="encrypt_device_text_for_profile_owner_setup" msgid="7828515754696057140">"Da biste podesili profil za posao, uređaj treba da se šifruje. To može da potraje."</string>
     <string name="encrypt_device_text_for_device_owner_setup" msgid="5194673142404735676">"Da biste podesili ovaj uređaj, prvo treba da se šifruje. To može da potraje."</string>
     <string name="encrypt_this_device_question" msgid="8719916619866892601">"Šifrovati ovaj uređaj?"</string>
     <string name="encrypt" msgid="1749320161747489212">"Šifruj"</string>
@@ -99,7 +99,7 @@
     <string name="profile_owner_cancel_ok" msgid="5951679183850766029">"Da"</string>
     <string name="profile_owner_cancelling" msgid="5679573829145112822">"Otkazuje se…"</string>
     <string name="work_profile_setup_later_title" msgid="9069148190226279892">"Želite li da zaustavite podešavanje profila?"</string>
-    <string name="work_profile_setup_later_message" msgid="122069011117225292">"Možete kasnije da podesite profil za Work u aplikaciji za upravljanje uređajem koju koristi organizacija."</string>
+    <string name="work_profile_setup_later_message" msgid="122069011117225292">"Možete kasnije da podesite profil za posao u aplikaciji za upravljanje uređajem koju koristi organizacija."</string>
     <string name="continue_button" msgid="7177918589510964446">"Nastavi"</string>
     <string name="work_profile_setup_stop" msgid="6772128629992514750">"Zaustavi"</string>
     <string name="dismiss" msgid="9009534756748565880">"Odbaci"</string>
diff --git a/res/values-de/strings.xml b/res/values-de/strings.xml
index c81e1f72..e515f645 100644
--- a/res/values-de/strings.xml
+++ b/res/values-de/strings.xml
@@ -54,7 +54,7 @@
     <string name="encrypt" msgid="1749320161747489212">"Verschlüsseln"</string>
     <string name="continue_provisioning_notify_title" msgid="5191449100153186648">"Verschlüsselung abgeschlossen"</string>
     <string name="continue_provisioning_notify_text" msgid="1066841819786425980">"Tippe, um die Einrichtung des Arbeitsprofils fortzusetzen"</string>
-    <string name="managed_provisioning_error_text" msgid="7063621174570680890">"Dein Arbeitsprofil konnte nicht eingerichtet werden. Bitte wende dich an deine IT-Abteilung oder versuche es später erneut."</string>
+    <string name="managed_provisioning_error_text" msgid="7063621174570680890">"Dein Arbeitsprofil konnte nicht eingerichtet werden. Bitte wende dich an deine IT-Abteilung oder versuche es später noch einmal."</string>
     <string name="cant_add_work_profile" msgid="9217268909964154934">"Arbeitsprofil kann nicht hinzugefügt werden"</string>
     <string name="cant_replace_or_remove_work_profile" msgid="7861054306792698290">"Arbeitsprofil kann nicht ersetzt oder entfernt werden"</string>
     <string name="work_profile_cant_be_added_contact_admin" msgid="4740182491195894659">"Diesem Gerät kann kein Arbeitsprofil hinzugefügt werden. Wende dich bei Fragen an deinen Administrator."</string>
diff --git a/res/values-mr/strings.xml b/res/values-mr/strings.xml
index c1ee7b35..ee91a21c 100644
--- a/res/values-mr/strings.xml
+++ b/res/values-mr/strings.xml
@@ -110,8 +110,8 @@
     <string name="device_owner_info" msgid="3716661456037934467">"हे डिव्हाइस <xliff:g id="YOUR_ORGANIZATION">%1$s</xliff:g> कडून व्यवस्थापित केले जाईल, त्याचे परीक्षण केले जाईल आणि ते सुरक्षित ठेवले जाईल. अटी लागू होतील. <xliff:g id="VIEW_TERMS">%2$s</xliff:g>"</string>
     <string name="device_owner_info_with_terms_headers" msgid="1254243288669282977">"हे डिव्हाइस <xliff:g id="YOUR_ORGANIZATION">%1$s</xliff:g> कडून व्यवस्थापित केले जाईल, त्याचे निरीक्षण केले जाईल आणि ते सुरक्षित ठेवले जाईल. <xliff:g id="TERMS_HEADERS">%2$s</xliff:g> च्या अटी लागू होतील. <xliff:g id="VIEW_TERMS">%3$s</xliff:g>"</string>
     <string name="link_isnt_secure_and_cant_be_opened_until_device_setup_finished" msgid="1604497932637832657">"ही लिंक सुरक्षित नाही आणि डिव्हाइस सेटअप पूर्ण होईपर्यंत उघडली जाऊ शकत नाही: <xliff:g id="LINK_RAW_TEST">%1$s</xliff:g>"</string>
-    <string name="contact_device_provider" msgid="9100405424740726066">"आपल्याला प्रश्न असल्यास, आपल्या <xliff:g id="ORGANIZATIONS_ADMIN">%1$s</xliff:g> शी संपर्क साधा."</string>
-    <string name="if_questions_contact_admin" msgid="6147462485780267795">"आपल्याला प्रश्न असल्यास, आपल्या संस्थेच्या प्रशासकाशी संपर्क साधा"</string>
+    <string name="contact_device_provider" msgid="9100405424740726066">"तुम्हाला प्रश्न असल्यास, आपल्या <xliff:g id="ORGANIZATIONS_ADMIN">%1$s</xliff:g> शी संपर्क साधा."</string>
+    <string name="if_questions_contact_admin" msgid="6147462485780267795">"तुम्हाला प्रश्न असल्यास, आपल्या संस्थेच्या प्रशासकाशी संपर्क साधा"</string>
     <string name="setup_isnt_finished_contact_admin" msgid="235011880559615998">"सेटअप पूर्ण झाला नाही. मदतीसाठी, आपल्या प्रशासकाशी संपर्क साधा."</string>
     <string name="for_help_contact_admin" msgid="8687037236275184653">"मदतीसाठी, आपल्या प्रशासकाशी संपर्क साधा"</string>
     <string name="organization_admin" msgid="1595001081906025683">"संस्थेचा प्रशासक"</string>
@@ -143,7 +143,7 @@
     <string name="expand" msgid="37188292156131304">"विस्तृत करा"</string>
     <string name="collapse" msgid="7817530505064432580">"संकुचित करा"</string>
     <string name="access_list_of_links" msgid="7094123315959323372">"दुव्यांच्या सूची अॅक्सेस करा"</string>
-    <string name="access_links" msgid="7991363727326168600">"प्रवेश दुवे"</string>
+    <string name="access_links" msgid="7991363727326168600">"प्रवेश लिंक"</string>
     <string name="access_terms" msgid="1982500872249763745">"अटी अॅक्सेस करा"</string>
     <string name="read_terms" msgid="1745011123626640728">"अटी वाचा"</string>
     <string name="close_list" msgid="9053538299788717597">"सूची बंद करा"</string>
diff --git a/res/values-pt-rPT/strings.xml b/res/values-pt-rPT/strings.xml
index 897717e3..eb53af01 100644
--- a/res/values-pt-rPT/strings.xml
+++ b/res/values-pt-rPT/strings.xml
@@ -70,7 +70,7 @@
     <string name="progress_connect_to_mobile_network" msgid="3852054186860657088">"A ligar à rede móvel…"</string>
     <string name="progress_download" msgid="3522436271691064624">"A transferir a aplicação de administração..."</string>
     <string name="progress_install" msgid="2258045670385866183">"A instalar a aplicação de administração..."</string>
-    <string name="progress_delete_non_required_apps" msgid="7633458399262691256">"A remover aplic. do sistema não necessárias…"</string>
+    <string name="progress_delete_non_required_apps" msgid="7633458399262691256">"A remover app do sistema não necessárias…"</string>
     <string name="progress_finishing_touches" msgid="9037776404089697198">"A dar os toques finais…"</string>
     <string name="progress_set_owner" msgid="8214062820093757961">"A definir o proprietário do dispositivo..."</string>
     <string name="progress_initialize" msgid="1104643492713424939">"A inicializar o dispositivo…"</string>
diff --git a/res/values-sr/strings.xml b/res/values-sr/strings.xml
index dd59bf3b..8406992c 100644
--- a/res/values-sr/strings.xml
+++ b/res/values-sr/strings.xml
@@ -48,7 +48,7 @@
     <string name="sure_you_want_to_delete_profile" msgid="6927697984573575564">"Ако наставите, избрисаћете све апликације и податке на овом профилу."</string>
     <string name="delete_profile" msgid="2299218578684663459">"Избриши"</string>
     <string name="cancel_delete_profile" msgid="5155447537894046036">"Откажи"</string>
-    <string name="encrypt_device_text_for_profile_owner_setup" msgid="7828515754696057140">"Да бисте подесили профил за Work, уређај треба да се шифрује. То може да потраје."</string>
+    <string name="encrypt_device_text_for_profile_owner_setup" msgid="7828515754696057140">"Да бисте подесили профил за посао, уређај треба да се шифрује. То може да потраје."</string>
     <string name="encrypt_device_text_for_device_owner_setup" msgid="5194673142404735676">"Да бисте подесили овај уређај, прво треба да се шифрује. То може да потраје."</string>
     <string name="encrypt_this_device_question" msgid="8719916619866892601">"Шифровати овај уређај?"</string>
     <string name="encrypt" msgid="1749320161747489212">"Шифруј"</string>
@@ -99,7 +99,7 @@
     <string name="profile_owner_cancel_ok" msgid="5951679183850766029">"Да"</string>
     <string name="profile_owner_cancelling" msgid="5679573829145112822">"Отказује се…"</string>
     <string name="work_profile_setup_later_title" msgid="9069148190226279892">"Желите ли да зауставите подешавање профила?"</string>
-    <string name="work_profile_setup_later_message" msgid="122069011117225292">"Можете касније да подесите профил за Work у апликацији за управљање уређајем коју користи организација."</string>
+    <string name="work_profile_setup_later_message" msgid="122069011117225292">"Можете касније да подесите профил за посао у апликацији за управљање уређајем коју користи организација."</string>
     <string name="continue_button" msgid="7177918589510964446">"Настави"</string>
     <string name="work_profile_setup_stop" msgid="6772128629992514750">"Заустави"</string>
     <string name="dismiss" msgid="9009534756748565880">"Одбаци"</string>
-- 
2.17.1

